monai/transforms/spatial/dictionary.py

```diff
@@ -756,7 +793,7 @@ def inverse(self, data: Mapping[Hashable, torch.Tensor]) -> dict[Hashable, torch
         return d
 
 
-class RandAffined(RandomizableTransform, MapTransform, InvertibleTransform):
+class RandAffined(RandomizableTransform, MapTransform, InvertibleTransform, LazyTransform):
```

Comment:
There may has some issues on this class, lazy resampling test passed on `RandAffine` but failed here (I used same input data and parameters to double check)